lib/floki/raw_html.ex

```diff
@@ -47,35 +47,59 @@ defmodule Floki.RawHTML do
         _ -> :noop
       end
 
-    IO.iodata_to_binary(build_raw_html(html_tree, [], encoder, padding))
+    self_closing_tags = MapSet.new(self_closing_tags())
```

Comment:
Just curious: did you find `MapSet` faster than using a `List` with unique elements?

Reply:
I did some tests, and it seems that performance is the same for this case, I reverted back to list 4d53e0f6e5f646e10a7d58d633015bd002338839